snap/hooks/install

```diff
@@ -30,16 +31,27 @@ fi
 chown -R 584788:root "${SNAP_COMMON}"/*
 
 # Copy over the mongod.conf and create needed directories/files
-echo "configuration file does not exist."
+echo "mongod configuration file does not exist."
 echo "copying default config to ${MONGO_CONFIG_FILE}"
 cp -r ${SNAP}/etc/mongod.conf ${MONGO_CONFIG_FILE}
 
+# Copy over the mongos.conf and create needed directories/files
+echo "mongos configuration file does not exist."
+echo "copying default config to ${MONGOS_CONFIG_FILE}"
+cp -r ${SNAP}/etc/mongod.conf ${MONGOS_CONFIG_FILE}
```

Comment:
Forgive me, I am forgetful. But on line 36 (`cp -r ${SNAP}/etc/mongod.conf ${MONGO_CONFIG_FILE}` ) we copy the original file mongod.conf, do you recall how that file was generated? I ask because I wonder if there is a pre-existing `mongos.py` file we could copy over.